docs/index.html

```diff
@@ -59,40 +59,40 @@ <h1 id="header-1">Features</h1>
                 <p>Now you won't have to anymore. This package allows you to use the latest version of those utility functions in all PHP_CodeSniffer versions from PHPCS 2.6.0 and up.</p>
             </dd>
 
-            <dt>A number of abstract sniff classes which your sniffs can extend.</dt>
+            <dt>Several abstract sniff classes which your sniffs can extend.</dt>
             <dd>
-                <p>These classes take some of the heavy lifting away for a number of frequently occuring sniff types.</p>
+                <p>These classes take some of the heavy lifting away for some frequently occurring sniff types.</p>
             </dd>
 
-            <dt>A collection of static properties for often used token groups.</dt>
+            <dt>A collection of static properties for often-used token groups.</dt>
             <dd>
-                <p>Collections of related tokens as often used and needed for sniffs.</br>
+                <p>Collections of related tokens often used and needed for sniffs.</br>
         These are additional "token groups" to compliment the ones available through the PHPCS native <span><code>PHP_CodeSniffer\Util\Tokens</code></span> class.</p>
             </dd>
 
-            <dt>An ever growing number of utility functions for use with PHP_CodeSniffer.</dt>
+            <dt>An ever-growing number of utility functions for use with PHP_CodeSniffer.</dt>
             <dd>
-                <p>Whether you need to split an <span><code>array</code></span> into the individual items, are trying to determine which variables are being assigned to in a <span><code>list()</code></span> or are figuring out whether a function has a docblock, PHPCSUtils got you covered!</p>
+                <p>Whether you need to split an <span><code>array</code></span> into the individual items, are trying to determine which variables are being assigned to in a <span><code>list()</code></span> or are figuring out whether a function has a DocBlock, PHPCSUtils has you covered!</p>
 
-                <p>Includes improved versions of the PHPCS native utility functions and plenty new utility functions.</p>
+                <p>Includes improved versions of the PHPCS native utility functions and plenty of new utility functions.</p>
 
                 <p>These functions are, of course, compatible with PHPCS 2.6.0 up to PHPCS <span><code>master</code></span>.</p>
             </dd>
 
             <dt>Test utilities</dt>
             <dd>
-                <p>An abstract <span><code>UtilityMethodTestCase</code></span> class to allow for testing your own utility methods written for PHP_CodeSniffer with ease.</br>
+                <p>An abstract <span><code>UtilityMethodTestCase</code></span> class to allow for easier testing of your utility methods written for PHP_CodeSniffer.</br>
```

Comment:
The new phrasing presumes that people are already unit testing their own utility methods, which may not be the case. WPCS being a case in point. The original phrasing did not make that assumption.

Reply:
Changed to `easy`.

Reply:
I try to stay away from saying something is `easy` as it can make people feel excluded.
And unit testing these utility functions still isn't "easy" for most people, but it is _easier_ with this testcase than it would otherwise be.

Reply:
Good point.

Reply:
Changed to:

> An abstract `UtilityMethodTestCase` class to support testing of your utility methods written for PHP_CodeSniffer.

To me, that neither implies something is easier, nor that they may or may not already have some tests - the testcase simply _supports_ them writing tests.

Reply:
Agreed. :+1: